src/ledger/test/LedgerTxnTests.cpp

```diff
@@ -1355,6 +1355,16 @@ TEST_CASE("LedgerTxn load", "[ledgertxn]")
                         NonSociRelatedException);
                 }
 
+                SECTION("invalid asset")
+                {
+                    std::string assetCode;
```

Comment:
Nitpick: this is a pretty weird way to write this. Why not just `auto asset = txtest::makeAsset(acc, "\n");`?

Reply:
Fixed